docs/how/customize-elasticsearch-query-template.md

```diff
@@ -0,0 +1,103 @@
+# How to customize the search experience of an entity?
+
+> Assume you have all the fields needed for your query, otherwise, refer to [this doc](./search-over-new-field.md) if you're only interested in indexing a new field of an existing entity 
+
+## 1. Default query template for each type of search document
+The search query is constructed and executed through [Search DAO]. Each search document is associated with a customizable query template, which can found [here](https://github.com/linkedin/datahub/tree/master/gms/impl/src/main/resources). 
+Take [dataset search query template] for example, it supports a few features:
+
+The general search accepts a query and at run time, [ES Search DAO] will replace $INPUT in the query template and generate the query for Elastic Search.
```

Comment:
s/Elastic Search/Elasticsearch/g

Reply:
Thanks, fixed the entire doc to keep consistent.